README.md

```
@@ -1,246 +1,69 @@
-
-## Quick Start
+| Package | Version | Description |
+|---------|---------|-------------|
+| [@rawsql-ts/ztd-cli](./packages/ztd-cli) | ![npm](https://img.shields.io/npm/v/@rawsql-ts/ztd-cli) | DB-agnostic scaffolding and DDL helpers for Zero Table Dependency projects. |
 
-### Installation
+## Architecture
 
-```bash
-npm install rawsql-ts
 ```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)